index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta http-equiv="X-UA-Compatible" content="ie=edge">
+		<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Sacramento&display=swap&subset=latin-ext">
+		<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300&display=swap">
+		<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+		<link rel="stylesheet" href="./css/app.css">
+		<title>The Library of Dracula</title>
+	</head>
+	<body>
+		<main>
+			<div class="container mt-5" id="root">
+        <h1 class="text-center py-5">Personal Catalogue</h1>
+        <div class="col-12 py-3 section-add-book" id="section-add-book">
```

Comment:
We are missing just one feature in this project 😞.
![image](https://user-images.githubusercontent.com/31897434/74258823-37802700-4cf7-11ea-8eb8-b4e18d562247.png)
https://www.theodinproject.com/courses/javascript/lessons/library

I guess you should have a `new book' button which the user can toggle on to display `form` for adding a new book.